Config/DefaultEngine.ini

```diff
@@ -47,3 +47,6 @@ ConnectionType=USBOnly
 bUseManualIPAddress=False
 ManualIPAddress=
 
+
+[CoreRedirects]
++FunctionRedirects=(OldName="/Script/UnrealExamples.WebLink.SetAvatarUrlCallback",NewName="/Script/UnrealExamples.WebLink.SetResponseCallback")
```

Comment:
This is usually necessary if someone else modifies the UnrealExamples project, and then tries to fetch the latest changes from git. I'd remove this, in theory, no one should modify the project and then update it.

Reply:
yep this was auto-added by rider I wasn't sure what it did :D will remove